src/web/components/MatchNav.tsx

```diff
@@ -31,11 +25,43 @@ const Row = ({ children }: { children: React.ReactNode }) => (
     </div>
 );
 
-const Container = ({ children }: { children: React.ReactNode }) => (
+const CrestRow = ({ children }: { children: React.ReactNode }) => (
     <div
         className={css`
-            background-color: ${brandYellow.main};
+            display: flex;
+            flex-direction: row;
+            align-items: flex-end;
+        `}
+    >
+        {children}
+    </div>
+);
+
+const StretchBackground = ({ children }: { children: React.ReactNode }) => (
+    <div
+        className={css`
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            position: relative;
             padding: ${space[2]}px;
+            /* We use min-height to help reduce our CLS value */
+            min-height: 230px;
+            background-color: ${brandYellow.main};
+
+            :before {
+                content: '';
+                position: absolute;
+                top: 0;
+                bottom: 0;
+                width: 100vw;
+                left: -100vw;
```

Comment:
I am curious of the use of `vw`? Is this done for parity?

Reply:
By using viewport width we can expand the background to cover the full viewport. This is very similar to how frontend does the same thing